src/main/resources/application.properties

```diff
@@ -2,11 +2,14 @@ server.port = 8090
 nrbestapi.version = ${NRBESTAPI_VERSION:#{'dev'}}
 server.allowed.cors.origins = ${ALLOWED_ORIGINS:#{'http://localhost:300*'}}
 
-spring.datasource.driver-class-name=oracle.jdbc.driver.OracleDriver
+spring.datasource.driver-class-name=oracle.jdbc.OracleDriver
 spring.jpa.database-platform=org.hibernate.dialect.Oracle12cDialect
 
 spring.datasource.url = jdbc:oracle:thin:@tcps://${DATABASE_HOST}:${DATABASE_PORT}/${SERVICE_NAME}
 spring.datasource.username = ${DATABASE_USER}
 spring.datasource.password = ${DATABASE_PASSWORD}
+spring.datasource.max-active=5
 
 management.endpoint.health.show-details = always
+
+hashicorp.vault.uri = ${VAULT_URI:#{'http://localhost:8888'}}
```

Comment:
By changing the dependency to the one mentioned above, it can be changed to the following properties:

```
spring.cloud.vault.authentication=TOKEN
spring.cloud.vault.token=tokenvalue
spring.cloud.vault.scheme=http
spring.cloud.vault.host=
spring.cloud.vault.port=
spring.cloud.vault.kv.enabled=true
spring.cloud.vault.kv.backend=secretsname
spring.config.import=vault://
```


This way you can use the secrets inside your properties/yml as:

```
spring.datasource.username=${secret.db.username}
```

This will keep the secret on vault on a KV secret with the following path: `secrets/kvname/appname` and you can use a dot notation key inside this path to be extracted on the yml/properties or using the `@Value` annotation or even a property class.